src/App.js

```diff
@@ -1,29 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/state-in-constructor */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] [ OPTIONAL]  KIndly fix linter issue manually after removing these lines. This is going mandatory requirement in further process.

Reply:
@Deepakdanger 
Thank you for your feedback am on it now